constants/Contributors.json

```diff
@@ -67,6 +67,11 @@
       "suffix": "§d:3"
     },
     "oxsss": {},
+    "PlasticEating": {
+      "suffix": "§d§l:3",
+      "spinny": true,

```

Comment:
please dont use spinny and upside down

Reply:
No Problem

Reply:
same with upside down

Reply:
Ahhh I just thought cuz a few others had it still